app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,35 @@
+<section class="signup_form">
```

Comment:
- Kindly follow the design guidelines for the Signup page:
   - [ ] Kindly add the `Back link` in the header which redirects to the previous page.
   - [ ] Please add the `Next link` instead of the `Sign out` link in the navbar in the header responsible for the sign-up process instead of adding a signup button.
   - [ ] Kindly remove the input's label and use placeholders instead.

| Your design | Guidelines |
| --- | --- |
| ![image](https://github.com/MNisarAli/budget_trackie/assets/24703171/2c306abc-4336-4f6e-a4c1-205545c58307) | ![image](https://user-images.githubusercontent.com/24703171/219884879-31b72a01-5fe0-4116-8fee-3a397873ebb9.png) |